exporter/elasticsearchexporter/README.md

```diff
@@ -513,7 +513,7 @@ The dimensions are mostly made up of resource attributes, scope attributes, scop
 
 The exporter can only group metrics with the same dimensions into the same document if they arrive in the same batch.
 To ensure metrics are not dropped even if they arrive in different batches in the exporter, the exporter adds a fingerprint of the metric names to the document in the `otel` mapping mode.
-Note that you'll need to be on a minimum version of Elasticsearch in order for this to take effect 8.16.5, 8.17.3, 8.19.0, 9.0.0.
+Note that you'll need to be on a minimum version of Elasticsearch in order for this to take effect 8.17.6, 8.19.0, 9.0.0.
```

Comment:
```suggestion
Note that this functionality requires a minimum Elasticsearch version of 8.17.6, 8.19.0, or 9.0.0.
```
This line sounds confusing to me. Maybe the above edit works?

Reply:
thanks!